### PR TITLE
disco rat

### PR DESCRIPTION
**Pull request by XWasHere**
woah

### CODE REVIEW

**Review by EmeraldSnorlax — CHANGES_REQUESTED** [review]
Could you add a seizure clickthrough?

**Comment by XWasHere** [comment]
good idea

**Comment by XWasHere** [comment]
> Could you add a seizure clickthrough?

Alright, I've updated it with a clickthrough.

File: hue/index.html [hunk 1]
```diff
@@ -10,10 +10,9 @@
   </head>
   <body>
     <img src="../bigrat.png" alt="An image of a rat sitting on a man's leg." id="rat">
-    <a>This page contains patterns which may trigger seizures. You have been warned
-Click the rat to continue
-
-(also, you can make it faster if you click multiple times, i didn't put a limit on it.</a>
+    <p>This page contains patterns which may trigger seizures. You have been warned
+    <p>Click the rat to continue
+    <p>(also, you can make it faster if you click multiple times, i didn't put a limit on it.</a>
```
Comment (EmeraldSnorlax) [inline hue/index.html]: the p tags arent closed, and the theres a hanging `</a>` tag

**Review by EmeraldSnorlax — CHANGES_REQUESTED** [review]

**Comment by XWasHere** [comment]
i think i fixed it... i need to fix my sleep schedule i'm falling apart

**Comment by EmeraldSnorlax** [comment]
when the sleep schedule is out of wack :flushed:

**Comment by XWasHere** [comment]
😳

**Review by EmeraldSnorlax — APPROVED** [review]
looks good enough to me :shrug:

**Pull request merged**